CHANGELOG.md

```diff
@@ -1,11 +1,10 @@
 Changes
 =======
 
-0.13.2, 2016-08-19
-* export_phrases in Phrases model changed. Fixed issue #794 and added test cases in test/test_phrases.py(@AadityaJ,
-[#879](https://github.com/RaRe-Technologies/gensim/pull/879))
```

Comment:
I didn't delete it, I had a previous build in which that entry was not in the changelog, anyway I added it. Thank You for marking out the changes.